web-common/src/main/java/edu/unc/lib/boxc/web/common/services/AccessCopiesService.java

```diff
@@ -137,6 +137,40 @@ public String getDatastreamPid(ContentObjectRecord briefObj, AccessGroupSet prin
         return null;
     }
 
+    /**
+     * Gets first object with an original file to use as thumbnail
+     * Returns null if briefObject has an original file, as we don't need to add the object twice.
+     * @param briefObj
+     * @param principals
+     * @return
+     */
+    public ContentObjectRecord getThumbnailObject(ContentObjectRecord briefObj, AccessGroupSet principals) {
+        if (permissionsHelper.hasOriginalAccess(principals, briefObj)) {
```

Comment:
Thumbnail access is based off of the viewMetadata permission rather than original access, you can see it here:
https://github.com/UNC-Libraries/box-c/blob/main/auth-api/src/main/java/edu/unc/lib/boxc/auth/api/services/DatastreamPermissionUtil.java#L48
And there's actually a helper method for checking thumbnail permissions specifically `permissionHelper.hasThumbnailAccess(principals, briefObj);`